spec/tapioca/cli/gem_spec.rb

```diff
@@ -48,6 +48,58 @@ def bar(a = T.unsafe(nil), b: T.unsafe(nil), **opts); end
 
       module Baz; end
 
+      class Baz::AbstractMethod
+        abstract!
+
+        def initialize(*args, &blk); end
+
+        sig { abstract.returns(String) }
+        def bar; end
+
+        sig { abstract.void }
+        def foo; end
+      end
+
+      class Baz::AbstractSingletonMethod
+        abstract!
+
+        def initialize(*args, &blk); end
+
+        class << self
+          sig { abstract.returns(String) }
+          def bar; end
+
+          sig { abstract.void }
+          def foo; end
+        end
+      end
+
+      class Baz::AbstractSingletonMethodAllNested
+        abstract!
+
+        class << self
+          sig { abstract.returns(String) }
+          def bar; end
+
+          sig { abstract.void }
+          def foo; end
+        end
+      end
```

Comment:
That might be because there are no methods in the regular class. I think this might be a bug. Maybe because Sorbet doesn't print any symbols for the regular class, Tapioca never realizes it needs to add an `initialize` method?

Reply:
The initialize is injected by sorbet-runtime when the class is declare abstract: https://github.com/sorbet/sorbet/blob/master/gems/sorbet-runtime/lib/types/private/abstract/declare.rb#L37-L42

This is why you see it appear only for abstract classes and not for the singleton.